### DISCUSSION TITLE
Welcome to gdsckwasu-hacktoberfest21 Discussions!

### DISCUSSION THREAD

**Discussion by kaf-lamed-beyt** [kwasu-ng/gdsckwasu-hacktoberfest21#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by benjaminob** [comment]
I'm Benjamin Babatola and I'm a Frontend developer.

**Comment by funabab** [comment]
Good day, for the design I'm proposing we go for tinder like design. :)

That's what immediately came to my head based on the current structure of details being displayed in card

**Reply by benjaminob** [reply]
Can we see a template?

**Reply by funabab** [reply]
[Tinder UI Design](https://duckduckgo.com/?q=tinder+ui+design&iax=images&ia=images)

**Reply by kaf-lamed-beyt** [reply]
Okay...

I guess we'll go with the Tinder UI then. Have you all considered the desktop UI? I mean... dunno you'd be building for desktop and mobile right?

**Reply by funabab** [reply]
That shouldn't be difficult, it's easier to adapt a mobile design for desktop.

That one can come later

**Comment by c-blaq** [comment]
Haven’t seen the desktop UI, don’t know if we can get a more detailed template

**Comment by funabab** [comment]
![hacktoberfest2021](https://user-images.githubusercontent.com/60202749/138424700-99b0b349-c602-4414-845f-70ac3b12ef06.png)
Brand for README banner

What do you guys thing?

**Reply by kaf-lamed-beyt** [reply]
Appears awesome boss ✊🏼

**Comment by kaf-lamed-beyt** [comment]
# Give us a Star 

Please, kindly give our Repo a star as you contribute to it
